app/src/main/java/com/example/kuba10/mypokemonplaces/AddPlaceFragment/AddPlaceFragment.java

```diff
@@ -1,169 +0,0 @@
-package com.example.kuba10.mypokemonplaces.AddPlaceFragment;
-
-
-import android.content.Context;
-import android.os.Bundle;
-import android.support.v4.app.Fragment;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.Button;
-import android.widget.ImageView;
-import android.widget.TextView;
-
-import com.example.kuba10.mypokemonplaces.ChooseFragment.ChooseFragment;
-import com.example.kuba10.mypokemonplaces.Constants;
-import com.example.kuba10.mypokemonplaces.FragmentListener;
-import com.example.kuba10.mypokemonplaces.Model.PokePlace;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.R;
-import com.google.android.gms.maps.model.LatLng;
-import com.squareup.picasso.Picasso;
-
-import java.util.ArrayList;
-import java.util.Date;
-
-import butterknife.BindView;
-import butterknife.ButterKnife;
-
-
-
-public class AddPlaceFragment extends Fragment {
-
-    private FragmentListener fragmentListener;
-    private ArrayList<PokemonGo> pokemonGo_data_list;
-    private PokemonGo selectedPokemon;
-
-    @BindView(R.id.pokemon_image_view)
-    ImageView pokemonImageView;
-    @BindView(R.id.description_input)
-    TextView descriptionField;
-    @BindView(R.id.title_input)
-    TextView titleField;
-    @BindView(R.id.add_button)
-    Button addButt;
-    @BindView(R.id.cancel_button)
-    Button cancelButt;
-
-    public static AddPlaceFragment newInstance(LatLng position) {
-        AddPlaceFragment fragment = new AddPlaceFragment();
-        Bundle args = new Bundle();
-        args.putParcelable(Constants.POSITION, position);
```

Comment:
Stała `Constants.POSITION` dotyczy bezpośrednio przerzucania argumentu w Fragmencie. Sugerowałbym umieszczenie tej stałej w klasie Fragmentu

Reply:
Nie wiem czemu uznalem ze jeden plik constants na caly projekt to lepszy pomysl - chyba jakis tutorial to sugerował. Jak to pisałem myślałem że cześciej będę przesyłał pozycje i globalna projektowa zmienna przyda sie w innych miejscach, ale w koncu wystarczyło przesłać raz.

Reply:
Tak, dużo programistów tworzy taki plik ale według mnie on jest bez sensu. Jeśli potrzebujesz nawet żeby jakaś stała była w kilku miejscach zrób po prostu ją `public`. Dzięki temu widać od razu czego ona dotyczy